src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -440,7 +440,7 @@ const PIAFormPage = () => {
       handleShowModal('edit');
     } else {
       navigate(
-        buildDynamicPath(pathname.replace('view', 'edit'), { id: pia.id }),
+        buildDynamicPath(pathname.replace('/view', '/edit'), { id: pia.id }),
```

Comment:
This was because it was replacing `view` in `/review/` with `edit`, making it `/reedit/`.